app/src/main/java/practicum/getfitla_v3/ExerciseItemFormat.java

```diff
@@ -1,46 +1,97 @@
 package practicum.getfitla_v3;
 
-import android.support.v7.app.AppCompatActivity;
+import android.os.Parcel;
+import android.os.Parcelable;
 
 //Formats Information to be easily called
-public class ExerciseItemFormat {
+public class ExerciseItemFormat implements Parcelable {
     private int id;
-    private String title;
+    private String name;
     private String shortdesc;
-    private String rating;
-    private String price;
+    private String isboolean;
+    private String equipment;
+    private String instructions;
     private int image;
 
-    public ExerciseItemFormat(int id, String title, String shortdesc, String rating, String price, int image) {
+
+    public ExerciseItemFormat(int id, String name,
+                              String shortdesc, String isboolean, String equipment, String instructions, int image) {
         this.id = id;
-        this.title = title;
+        this.name = name;
         this.shortdesc = shortdesc;
-        this.rating = rating;
-        this.price = price;
+        this.isboolean = isboolean;
+        this.equipment = equipment;
+        this.instructions = instructions;
         this.image = image;
+
     }
 
     public int getId() {
```

Comment:
I made this comment on Dan's code, but I'm not sure why you need all these methods. Can't you just get the objects attributes by doing: 
```
int id = exampleObject.id;
```
I guess I don't see the point in writing another method to do the same thing

Reply:
Ah ok, I wasn't familiar with this, but this is fine!